vars/all_var.yml

```diff
@@ -210,56 +92,12 @@
   db_user_name: "{{ env_db_user_name }}"
   db_user_pw: "{{ env_db_user_pw }}"
   db_platform_type: "postgres"
-  db_engine_driver: "django.db.backends.postgresql_psycopg2"
   db_host_address: "db.bluebutton-{{ env }}.local"
   db_host_port: "15432"
   db_host_options: "?sslmode=require&options=-c role={{ env_db_role }}"
 
-  # Database SSL Certificate and Key Files:
-  ca_key: "server.key"
-  ca_cert: "server.crt"
-
+  # legacy value still used in roles, should be renamed for clarity
   cf_region: "{{ aws_region }}"
```

Comment:
Unfortunately this variable is still used in playbooks/roles, opted to leave it and not refactor for now.